### ISSUE TITLE
Classical Molecular Networking fails with a subset of MASSIVES

### ISSUE THREAD

**Issue by mabelgon88** [CCMS-UCSD/GNPS_Workflows#391]
Hi Ming,

I am running a group of different MASSIVES in different jobs. One with 9 MASSIVES and another with 10 MASSIVES. The one with 10 MASSIVES runs perfect (https://gnps.ucsd.edu/ProteoSAFe/status.jsp?task=f49d235de130423392e878b8a158969d), but another with 9 MASSIVES failed even repeating the process (https://gnps.ucsd.edu/ProteoSAFe/status.jsp?task=14fab41ea2c140eeb98f4ab429620c35, https://gnps.ucsd.edu/ProteoSAFe/status.jsp?task=9ef6a2af2bce4afe853fc04ae7154e52).

I found this really weird because I used the 10 MASSIVE job and clone it, removing one of the MASSIVE files for running the job with 9 MASSIVES.

Thanks a lot for your help.

**Comment by mwang87** [comment]
THere is likely a bad file in the ones you selected. Try to remove that ones that are bad and go from there.

**Comment by mabelgon88** [comment]
I don't think so, because it is what I told you I started cloning the job with 10 MASSIVES that it works, and from there I removed one MASSIVE.

**Comment by mwang87** [comment]
The error seems to indicate otherwise: 

```Assertion `fabs(cluster->clusterMOverZ_-singletons[0]->getHeader()->getMOverZ())<=10.0' failed.```

seems to point that there is a spectrum with an mz less than 10.

**Comment by mabelgon88** [comment]
There is no chance that something weird is happening? Because I have the same issue with other jobs, I re-run it and mysteriously the second time works fine. 

Failed: https://gnps.ucsd.edu/ProteoSAFe/status.jsp?task=a64e1cde47fe457ea92754f6249fd780
Done: https://gnps.ucsd.edu/ProteoSAFe/status.jsp?task=58823863ff0c47cf9c755295a59a1201

**Comment by mwang87** [comment]
Thanks! Can you just do one check to make sure the parameters are identical?

**Comment by mwang87** [comment]
No response on issue. Closing.

**Comment by mabelgon88** [comment]
https://gnps.ucsd.edu/ProteoSAFe/status.jsp?task=14fab41ea2c140eeb98f4ab429620c35

**Comment by mabelgon88** [comment]
Hi Ming

This is one of the jobs that is still faling: https://gnps.ucsd.edu/ProteoSAFe/status.jsp?task=6c9aa617aa8e47c391af4813f4b0d885

When I split it is running ok. 

This are the links of the splited MASSIVES

https://gnps.ucsd.edu/ProteoSAFe/status.jsp?task=e138f8085a7846f9a33de5e35f03bc6d
https://gnps.ucsd.edu/ProteoSAFe/index.jsp?task=c0594dc7d076455a92d5c11c99a73cf0

So, what do you think that it could be the next logical step?

**Comment by mwang87** [comment]
The failure is because the data you put in is too large. You will need to increase the minimum cluster size.
